src/store/model/store.ts

```diff
@@ -1,8 +1,8 @@
 import {Browser} from 'puppeteer';
 
 export interface Link {
-	series: string;
-	brand: string;
+	brand: 'TEST' | 'asus' | 'evga' | 'gigabyte' | 'pny' | 'msi' | 'nvidia' | 'zotac';
+	series: 'debug' | '3070' | '3080' | '3090';
```

Comment:
I think we should settle on a brand/series for debug/test. I honestly don't know if either have any functional implications, but my guess no. Do you mind changing to this?

```suggestion
	brand: 'test:brand' | 'asus' | 'evga' | 'gigabyte' | 'pny' | 'msi' | 'nvidia' | 'zotac';
	series: 'test:series' | '3070' | '3080' | '3090';
```

Also, was there any more elaboration on:

> Opted not to use enums because the code itself is fine how it is.

If you wanted to keep this PR to a minimal, that makes sense. Just seeing if there was any technical reason you say as well.

Anyway, thanks for the contribution! I appreciate it!

Reply:
Yeah just keeping the PR minimal